packages/atlas/src/hooks/useMsTimestamp.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react'
+
+export const useMsTimestamp = () => {
+  const [timestamp, setTimestamp] = useState(Date.now())
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimestamp(Date.now())
+    }, 1000)
+
+    return () => clearInterval(interval)
+  }, [])
+
+  return timestamp
+}
```

Comment:
Maybe we could name this differently, just `useTimestamp` and pass to it parameter which could be `ms`, `second`, maybe even `minutes`? 
Something like 
```tsx
 const msTimestamp = useTimestamp('ms')
 const secTimestamp = useTimestamp('second')
//...
```

Reply:
Do we have a good use case for using different unit? I think it may be easier to maintain if we just pick one (`ms`) and stick with it

Reply:
ok, fair point 👍🏻